crates/solver/src/settlement/settlement_encoder.rs

```diff
@@ -139,14 +135,14 @@ impl SettlementEncoder {
             .any(|(_, internalizable)| !internalizable)
     }
 
-    // Fails if any used token doesn't have a price or if executed amount is impossible.
-    pub fn add_trade(
+    /// Adds an order trade using the uniform clearing prices for sell and buy token.
+    /// Fails if any used token doesn't have a price or if executed amount is impossible.
+    fn add_ucp_trade(
```

Comment:
Personally I wouldn't use the `ucp` acronym. Since `OrderClass::Ordinary` will be renamed to `OrderClass::Market`, why not call this `add_market_trade`? And in general other places where the term `ucp` is used can be changed to `market` probably.

Reply:
I also slightly prefer `add_market_trade`

Reply:
Added in [04be9dd](https://github.com/cowprotocol/services/pull/712/commits/04be9dddc85cfb03ec65059742b7ebf75459b444).